package.json

```diff
@@ -84,7 +85,7 @@
     "release:pr": "dripip pr",
     "format": "prettier --write '*/**/*.ts'",
     "build": "yarn -s clean && tsc -b",
-    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "clean": "rimraf dist tsconfig.tsbuildinfo",
```

Comment:
Just this change could be its own PR if you want to keep things rolling. Also think its worth started a scripts folder where we put all of these. Then we can use `fs-jetpack`. Again its what we use heavily. Would like to avoid deps where practical.